### ISSUE TITLE
rule: table-no-padding

### ISSUE THREAD

**Issue by purupurupu** [purupurupu/css-noop-checker#124]
## Rule ID
`table-no-padding`

## What does this rule detect?
Padding does not apply to `table-row`, `table-row-group`, `table-header-group`, `table-footer-group`, `table-column`, `table-column-group`. It only works on `table-cell`, `table`, `inline-table`, and `table-caption`.

## Example: should warn
```html
<table>
  <tr style="padding: 10px;"><td>cell</td></tr>
</table>
```

## Example: should NOT warn
```html
<table>
  <tr><td style="padding: 10px;">cell</td></tr>
</table>
```

## Related CSS Properties
`display`, `padding`, `padding-top`, `padding-right`, `padding-bottom`, `padding-left` (and logical equivalents)

## References
- [CSS 2.2 §17.5 - Visual layout of table contents](https://www.w3.org/TR/CSS22/tables.html#table-display): padding does not apply to rows, row groups, columns, column groups
- [Firefox inactive-property-helper.js](https://searchfox.org/mozilla-central/source/devtools/server/actors/utils/inactive-property-helper.js) — detects this pattern
